dev/InfoBar/InfoBar_themeresources.xaml

```diff
@@ -7,10 +7,10 @@
 
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Light">
-            <SolidColorBrush x:Key="InfoBarErrorSeverityBackgroundBrush" Color="#FDE7E9"/>
-            <SolidColorBrush x:Key="InfoBarWarningSeverityBackgroundBrush" Color="#FFF4CE"/>
-            <SolidColorBrush x:Key="InfoBarSuccessSeverityBackgroundBrush" Color="#DFF6DD"/>
-            <SolidColorBrush x:Key="InfoBarInformationalSeverityBackgroundBrush" Color="#F2F2F2"/>
+            <StaticResource x:Key="InfoBarErrorSeverityBackgroundBrush" ResourceKey="SystemFillColorCriticalBackgroundBrush"/>
+            <StaticResource x:Key="InfoBarWarningSeverityBackgroundBrush" ResourceKey="SystemFillColorCautionBackgroundBrush"/>
+            <StaticResource x:Key="InfoBarSuccessSeverityBackgroundBrush" ResourceKey="SystemFillColorSuccessBackgroundBrush"/>
+            <StaticResource x:Key="InfoBarInformationalSeverityBackgroundBrush" ResourceKey="SystemFillColorAttentionBackgroundBrush"/>
```

Comment:
Nit: to match global WinUI style syntax, add single space before all closing tags

Reply:
That is a pattern? I don't think there is a strong favor for either of those spacing styles.

Reply:
When I look at generic.xaml, almost all closing tags have a leading space which makes me think it was a deliberate decision to make it the style.

Reply:
I guess this wasn't brought forward to the WinUI code base then.

Reply:
It's kind of random. I see it both ways. But for this file... sure. :)